ui/package.json

```diff
@@ -12,28 +12,34 @@
     "lint": "eslint --fix --ext .ts ./ --ext .vue src"
   },
   "dependencies": {
-    "@holochain-open-dev/profiles": "^0.12.4",
-    "@holochain/client": "^0.12.3",
-    "@quasar/extras": "^1.15.4",
-    "@shoelace-style/shoelace": "^2.3.0",
+    "@holochain-open-dev/profiles": "^0.14.1",
+    "@holochain/client": "^0.14.1",
+    "@shoelace-style/shoelace": "^2.4.0",
+    "async-retry": "^1.3.3",
     "dayjs": "^1.11.7",
     "lodash.flatten": "^4.4.0",
+    "lodash.isequal": "^4.5.0",
```

Comment:
Regarding using particular modules of lodash, you could switch to the more future-proof way of importing by installing the whole lodash package and then import
```javascript
import array from "lodash/array";
```

The per-method packages will be removed in v5 https://lodash.com/per-method-packages

Reply:
done